style.css

```diff
@@ -29,7 +33,7 @@ nav {
 
 .primary-navigation {
   display: flex;
```

Comment:
can be clubbed into a class, as it is used many times

Reply:
Thank you for pointing it! Overlooked this